agent/lib/docker/container.rb

```diff
@@ -34,6 +34,12 @@ def restart_policy
       self.host_config['RestartPolicy']
     end
 
+    # @return [String] Stack aware name of the containers service
+    def service_name
+      return self.labels['io.kontena.service.name'] if self.default_stack?
```

Comment:
It can't use `/` as separator, LB is listing plain values under `/kontena/haproxy/<lb service>/services` and having `/` as separator breaks that.

This is currently used only for this LB configs, so I'll rename it to reflect that.